src/locales/cy/translation.json

```diff
@@ -500,7 +500,8 @@
     },
     "securityCodeWaitToRequest": {
       "title": "Ni allwch gael cod diogelwch newydd ar hyn o bryd ",
-      "header": "Ni allwch gael cod diogelwch newydd ar hyn o bryd ",
+      "header_old": "Ni allwch gael cod diogelwch newydd ar hyn o bryd ",
```

Comment:
I'm a bit confused by some of the key suffixes used for `header` and `info` here, i.e. `_old` and `_create`. The values seem to suggest they're different things and I'm wondering if these differences should be represented in the keys (in the way that `header_create` is, for example)

Reply:
For this, the old values are still used when the flag is turned off. This will be used in more than one user journey, so I assume it's ok to leave it like this since it indicates that it corresponds to the old lockout pages.